app/models/ticket_purchase.rb

```diff
@@ -52,7 +52,8 @@ def self.purchase_ticket(conference, quantity, ticket, user)
       purchase = new(ticket_id: ticket.id,
                      conference_id: conference.id,
                      user_id: user.id,
-                     quantity: quantity)
+                     quantity: quantity,
+                     amount_paid: ticket.price)
```

Comment:
what about calling this `price`? :thinking:  In our discussion this make sense to diferenciate this from the ticket price, but I doubt that this is easily understandable by someone else outside the discussion when reading this in some months.

Reply:
I think calling it price is better, but i used amount_paid so that others can differentiate it from ticket price.I will change it to price.